src/main/java/com/demo/classroom/Entity/Book.java

```diff
@@ -0,0 +1,37 @@
+package com.demo.classroom.Entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(updatable = false, nullable = false)
+    private Long id;
+
+    @NotBlank(message = "Book must have a name.")
+    private String name;
```

Comment:
Use consistent validation

Reply:
Added.

Reply:
`@NotBlank` it provides application/UI level validation. It ensures Not null, No Empty or whitespace is allowed. 

`@NotBlank` ensures that no null value is going to have in my database in that particular column.